static/app/components/workflowEngine/form/control/priorityControl.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import {act, render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import PriorityControl from 'sentry/components/workflowEngine/form/control/priorityControl';
+import {PriorityLevel} from 'sentry/types/group';
+
+describe('PriorityControl', function () {
+  it('renders children', async function () {
+    render(<PriorityControl name="priority" />);
+
+    expect(await screen.findByText('Issue created')).toBeInTheDocument();
+    expect(await screen.findByTestId('priority-control-medium')).toBeInTheDocument();
+    expect(await screen.findByTestId('priority-control-high')).toBeInTheDocument();
+  });
+  it('allows configuring priority', async function () {
+    const mock = jest.fn();
+    act(() => {
+      render(
+        <PriorityControl
+          priority={PriorityLevel.LOW}
+          onPriorityChange={mock}
+          name="priority"
+        />
+      );
+    });
+    await userEvent.click(screen.getByRole('button'));
```

Comment:
you shouldn't need to wrap render in act. try  starting with a findBy like the other tests

```suggestion
      render(
        <PriorityControl
          priority={PriorityLevel.LOW}
          onPriorityChange={mock}
          name="priority"
        />
      );
    await userEvent.click(await screen.findByRole('button'));
```